.github/workflows/ci.yml

```diff
@@ -64,22 +64,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ "macos-12", "macos-14", "windows-latest", "ubuntu-latest" ]
-        python-version: [ "3.9", "3.12", "3.13-dev" ]
+        platform: [ "macos-13", "macos-latest", "windows-latest", "ubuntu-latest" ]
+        python-version: [ "3.9", "3.13" ]
         include:
           - experimental: false
-          - platform: "ubuntu-latest"
-            python-version: "3.9"
-            experimental: false
+          # Ensure the Python versions between min and max are tested
           - platform: "ubuntu-latest"
             python-version: "3.10"
             experimental: false
           - platform: "ubuntu-latest"
             python-version: "3.11"
             experimental: false
-          # Allow dev Python to fail without failing entire job
-          - python-version: "3.13-dev"
-            experimental: true
+          - platform: "ubuntu-latest"
+            python-version: "3.12"
+            experimental: false
+          # # Allow dev Python to fail without failing entire job
+          # - python-version: "3.14"
+          #   experimental: true
```

Comment:
I was thinking...is it possible to default `experimental` to `false` so it only has to be set for dev Python?

For instance, maybe:
```yaml
continue-on-error: ${{ matrix.experimental || false }}
```
or perhaps:
```yaml
continue-on-error: ${{ matrix.experimental && true || false }}
```

I think originally `continue-on-error` did not support arbitrary expressions so everyone defaulted a value in their matrix config.